data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			`	"What is it you need to know, officer?" the captain asks. Although Xilin isn't a Navy officer, he doesn't correct her.`
+			`	"I'd like to know why it is that you decide to bring all these passengers to Hai space who had never been here before."`
+			`	The captain scratches the back of her head. "Well after seeing the news back home, I figured the secret was already out. Might as well make the most of it, you know?"`
+			choice
+				`	(Let Xilin handle this.)`
+					goto xilin
+				`	"Didn't you hear the Navy's advisory? You could have gotten your passengers killed coming here."`
+			`	The captain scoffs. "Well yeah, but I figured it was just a coverup from the Navy to stop people from coming here."`
+			`	"Would that not imply that the secret isn't truly 'out' then?" Sayari asks while giving the captain a stern look.`
+				goto trouble
+			label xilin
+			`	"No, I don't know," says Xilin while giving the captain a stern look. "Rumors of the Hai becoming mainstream don't constitute the secret being 'out' to me."`
+			label trouble
+			`	"Well when you put it that way," the captain says nervously. "Ok, just spit it out already. Am I in trouble?"`
+			`	"No," Xilin says while looking away. Something else has caught his attention. "But they might be."`
+			`	You look up to see a beat-up Berserker that looks like it can barely fly landing in the spaceport. "You're free to go," Xilin says to the Star Queen captain.`
+			`	You approach the Berserker as it finishes landing. The captain of the ship exits looking far calmer than the condition of the Berserker would suggest its captain should be. "Uh, hi," he says akwardly. "Can I help you guys?"`
+			choice
+				`	"What happened to your ship?"`
+				`	"Did you get attacked by pirates on the way here."`
+			`	The captain looks back at the poor state of his interceptor. "Oh, that. Yeah, I ran into a couple pirates on the way here. Nothing I haven't dealt with before, though."`
+			`	"So you've been to Hai space before?" Xilin asks.`
+			`	"Sure. Plenty of times," he responds.`
+			`	Xilin looks over the interceptor with a quizzical expression. "May I see your pilot's license?"`
+			`	"Yeah, no problem," the captain says, as he quickly pulls a license out of his pocket that looks almost as beat up as the interceptor. Xilin jots down the details and hands the license back to the captain. "Thank you. And maybe tag along with a merchant fleet on your way back home."`
+			`	"That sounds like a good idea," the captain says before walking off into the spaceport.`
+			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Some already knew of the Hai before, but many are new arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains. Everyone seems peaceful enough in interacting with the Hai, but clearly the word is getting out more as time goes on.`
```

Comment:
In line with not quite so many daily arrivals, adjusting the acquisition of information - and also planting someone that could arguably be counted as an ongoing informant for Xilin to handwave acquiring updated data in the future.
```suggestion
			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Most turn out to be repeat visitors or those who have taken to conducting their business entirely in Hai space, but they are all familiar with arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains and many are able to provide information about these encounters. One captain in particular has been camped out here just intercepting new captains as they arrive and giving them the induction tour, and has a copious set of notes that he gladly shares with Xilin. It seems that everyone has been peaceful enough in interacting with the Hai so far, but clearly the word is getting out more as time goes on.`
```

Reply:
Ditto.